dev-docs/bidders/viewdeosDX.md

```diff
@@ -14,8 +14,30 @@ gdpr_supported: true
 | Name  | Scope    | Description                     | Example  | Type      |
 |-------|----------|---------------------------------|----------|-----------|
 | `aid` | required | The source ID from member zone | `350975` | `integer` |
+| `outstream` | optional | Oustream player settings object | `{"audio_setting":"on","video_controls":"show"}` | `object` |
 
+#### Oustream settings object
 
+{: .table .table-bordered .table-striped }
+| Name  | Scope    | Description                     | Possible Values  | Type      |
+|-------|----------|---------------------------------|----------|-----------|
+| `type` | optional | allows to set the outstream type  | `inread/inbanner/interstitial/slider` | `string` |
```

Comment:
So for example, the Description here would be "Set the outstream type to inread/inbanner/interstitial/slider." The Example could be 'inbanner'.

Reply:
Fixed